src/patternfly/components/Table/table.scss

```diff
@@ -638,26 +643,65 @@
       --pf-c-table__compound-expansion-toggle__button--before--BorderLeftWidth: 0;
     }
   }
+
+  &:active,
+  &:focus-within {
+    outline-offset: var(--pf-c-table__button--OutlineOffset);
+
+    // stylelint-disable media-feature-name-no-vendor-prefix
+    @media (-webkit-min-device-pixel-ratio: 0) {
+      outline-style: auto;
+      outline-color: -webkit-focus-ring-color;
+    }
+    // stylelint-enable
+  }
+}
+
+.pf-c-table__column-help {
+  display: flex;
+  align-items: flex-end;
```

Comment:
is this needed

Reply:
Yeah, it keeps everything bottom aligned in the cell.

Here's with

<img width="896" alt="Screen Shot 2020-06-18 at 10 54 54 AM" src="https://user-images.githubusercontent.com/35148959/85043673-3bfb9700-b152-11ea-8f56-0250169459a4.png">

Here's without

<img width="874" alt="Screen Shot 2020-06-18 at 10 55 03 AM" src="https://user-images.githubusercontent.com/35148959/85043697-4322a500-b152-11ea-8463-effbf2c73059.png">